oaknational/Oak-Web-Application
# feat(printable): create style printable results page

## Description

Music year: 2006

- List of changes
PUPIL-967 - printable results page 

now when a pupil finishes a lesson and presses get printable results they're results are stored locally and they can see the printable results page

- owa points to pupil-api prod
- video completion percentage is accurate

### Acceptance criteria

- [x]  pupils/results/print/[lessonSlug]/[attemptId]
- [x]  fetches results from localStorage
- [x]  loading state

PUPIL-1033 - styling
I have piggy backed the styling ticket for results page here; so both share page and printable page have mikes styling.

figma
https://www.figma.com/design/G7z5ufo2T0CHk14CkadeuB/Pupil-Lesson?node-id=10015-27969&node-type=canvas&t=OwKu20WMfailZlrk-0
## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2810--oak-web-application.netlify.thenational.academy
2. Click on a lesson; finish a lesson; on review page you should see two buttons; pritable and share; printable should take you to the styled printable page and share after loading will give you a url of the styled share page

note. you need to change the url from "thenational.academy" to "https://deploy-preview-2810--oak-web-application.netlify.thenational.academy"

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
